docs/overview.rst

```diff
@@ -102,3 +102,65 @@ For example, the `Escrow` contract factory requires linking to an instance of th
        deployer.register_strategy("Escrow", escrow_strategy)
        linked_escrow_package, _ = deployer.deploy("Escrow") 
        return linked_escrow_package.deployments.get_deployment("Escrow")
+
+
+Log
+---
+
+The ``Log`` class is available to help with testing for contract events, and the contents of the emitted logs.
+
+.. code:: python
+
+   # SETUP
+   ping_contract = """
+   Ping: event({_first: indexed(bytes32), _second: bytes32})
+
+   @public
+   def __init__():
+       pass
+
+   @public
+   def ping(first: bytes32, second: bytes32):
```

Comment:
Instead of inlining a copy of the source here, what about inlining the actual fixture file so that the two stay synced up.

http://docutils.sourceforge.net/docs/ref/rst/directives.html#include